src/index.html

```diff
@@ -5,9 +5,68 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <h1></h1>
+    <div
+      class="product">
+      <div
+        class="product-card"
+        data-qa="card"
+        >
+        <div class="product-image-container">
+          <img
+              class="product-image"
+              src="../src/images/imac.jpeg"
+              alt="image"
+            >
+        </div>
+
+        <h2 class="product-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-code">
+            Product code: 195434
+        </p>
```

Comment:
Need to fix it everywhere
```suggestion
        <p class="product-code">
          Product code: 195434
        </p>
```